lib/src/dartdoc/dartdoc_options.dart

```diff
@@ -61,5 +61,6 @@ final _passThroughKeys = <String>[
   // 'examplePathPrefix',
   'exclude',
   'include',
+  'showUndocumentedCategories',
```

Comment:
Note: we do have `'showUndocumentedCategories': true` on line 52. Isn't that enough?

Reply:
Yes, that's should be enough, no idea why it's not working.